packages/common/src/store/cache/collections/utils/index.ts

```diff
@@ -0,0 +1,3 @@
+// TODO: Migrate utils from packages/web/common/store/cache/collections/utils to here
```

Comment:
Is this a leftover comment or something for the future? If it's for the future, do you mind making a linear item and adding the link here?

Reply:
Yess. Basically the task is to finish the common migration, will update this comment. https://linear.app/audius/issue/C-1280/migrate-remaining-webcommon-to-common